File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        
        int[] ans = new int[2];
        int n = nums.length;
        for(int i=0;i<n;i++){
            int remainingSum = target - nums[i];
            for(int j=i+1;j<n;j++){
                if(remainingSum == nums[j]){
                    return new int[] { i, j };
                }
            }

        }
        return new int[] {};
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        diction = dict()
        for i in range(len(nums)):
            diction[nums[i]] = i 

        for i in range(len(nums)):
            diff = target - nums[i]

            if diff in diction and diction[diff] != i:
                return [i, diction[diff]]
        return []


        